src/controllers/TorusController.ts

```diff
@@ -1354,6 +1356,13 @@ export default class TorusController extends BaseController<TorusControllerConfi
     end();
   }
 
+  private async loginWithPrivateKey(req: IEhandler<LoginWithPrivateKeyParams>): Promise<{ success: boolean }> {
+    if (!req.params?.privateKey) throw new Error("Invalid Private Key");
+    const address = await this.addAccount(req.params?.privateKey, req.params?.userInfo);
```

Comment:
you need to toggle widget status in embed once the user is logged in (is it happening already).

Reply:
Yes, by default the widget is visible once user is login (has keypair and selectedAddress)
unless is set hidden ( isIframeFullScreen = false ) by embed 
Current Solana embed default setting is set hidden